### PR TITLE
Add cht.sh Go Cheatsheet entry

### PR DESCRIPTION
**Pull request by RolloCasanova**
A quick tutorial/reference to learn Go.

## What does this PR do?
Add resource

### Description
 
Add entry to cht.sh go tutorial.

### Why is this valuable (or not)?

It allows you to quickly learn about Go programming language syntaxis, common examples of how to perform basic stuff with the language.

### How do we know it's really free?

It's a free webpage; you can even download the cht.sh client on your computer.

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/master/CONTRIBUTING.md)
- [x] Search for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction)

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -333,6 +333,7 @@
 ### Go
 
 * [A Tour Of Go](https://tour.golang.org/welcome/1)
+* [cht.sh Go Cheatsheet](https://cht.sh/go/:learn)
```
Comment (davorpa) [inline courses/free-courses-en.md:336]: :information_source: I'm almost sure that if is a cheatsheet it needs be moved to right listing `free-programming-cheatsheets.md` ([in your branch](https://github.com/RolloCasanova/free-programming-books/blob/cht.sh/more/free-programming-cheatsheets.md))

:question: Why `go:learn` and not merelly `go`?

**Review by davorpa — CHANGES_REQUESTED** [review]

File: courses/free-courses-en.md (hunk 1 above)
Comment (RolloCasanova) [reply courses/free-courses-en.md:336]: Well, I thought `/go:learn` was kind of a quick course; will move it to `free-programming-cheatsheets.md`.

`/go` just shows basic commands for Go in CLI.

File: more/free-programming-cheatsheets.md [hunk 2]
```diff
@@ -68,6 +69,11 @@
 * [GitHub Cheat Sheet](https://github.com/tiimgreen/github-cheat-sheet) - Tim Green (Markdown)
 
 
+### Go
```
Comment (davorpa) [inline more/free-programming-cheatsheets.md:72]: :information_source: It's usually not helpful to add a category for a single resource. Try to find another category, or see if there are other books in our lists that would fit into your category.

**Review by davorpa — APPROVED** [review]
### Ready to merge :fireworks: :tada:

I'm "go"ing to be a gopher :hugs:. It is a good point to have this section as cheatsheet

**Comment by eshellman** [comment]
It's a go. 
Thanks, and welcome to free-programming-books!

**Pull request merged**